packages/edit-site/src/components/global-styles/screen-revisions/style.scss

```diff
@@ -78,20 +96,26 @@
 	flex-direction: column;
 	align-items: flex-start;
 	gap: $grid-unit-10;
+	.edit-site-global-styles-screen-revisions__date {
+		text-transform: uppercase;
+		font-weight: 600;
+		font-size: 12px;
```

Comment:
For each of the `font-size: 12px` rules, should we reference `$helptext-font-size` instead?

Reply:
I saw that but wasn't sure whether I should use it as it's specific to help text and might change?

We could add another var I guess 🤔

Reply:
Feel free to leave it, then, it's only two hard-coded lines 🙂